pilz_store_positions/README.md

```diff
@@ -1,14 +1,47 @@
 # Store poses during teaching
 
-After moving the robot to the desired position, you can store the current position to file.
+## Preparation
+
+To prepare for saving position you have to specify a file.
+Open a terminal and enter:
+
+```
+roslaunch teach_positions.launch [_file_path := "/path/to/file.py"]
```

Comment:
Why trailing underscore?

Reply:
Maybe you mixed up with `rosrun` which [uses underscores](https://answers.ros.org/question/299014/command-line-argument-passing-in-rosrun/?answer=299124#post-id-299124) for parameters? For `roslaunch` you could drop the underscore imho.